src/core/label/manager.go

```diff
@@ -107,8 +112,8 @@ func (bm *BaseManager) GetLabelsOfResource(resourceType string, resourceIDOrName
 }
 
 // Exists is the implementation of same method in Manager interface.
-func (bm *BaseManager) Exists(labelID int64) (*models.Label, error) {
-	label, err := dao.GetLabel(labelID)
+func (bm *BaseManager) Exists(labelID int64) (*model.Label, error) {
+	label, err := bm.LabelMgr.Get(orm.Context(), labelID)
```

Comment:
If the label doesn't exist, will a not found error be returned? If it is, should replace the logic:
```
if label == nil {
	return nil, NewErrLabelNotFound(labelID, "", nil)
}
```

Reply:
updated.